### DISCUSSION TITLE
如何让Alas在计算机启动后自动执行多个配置文件？

### DISCUSSION THREAD

**Discussion by HS7097** [LmeSzinc/AzurLaneAutoScript#2128] [Q&A] [answered]
book大佬的bat文件我在使用的时候只能直接启动一个配置文件，且，启动后config文件夹内新创建的reloadalas会被直接删除；输入配置文件名+.maa到新建的文件后再通过bat文件启动也没有响应
有没有大佬提供一个自动化的方案，谢谢啦qwq

**Comment by bookbug666** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/29662367/212667401-f4e00202-d9f7-4104-8325-790e435f84b1.png)
現在已經改用這個一段時間，比較簡單不用折騰batch了
最底下填入你的config name，在./config/deploy.yaml裡面

**Reply by HS7097** [reply]
谢谢大佬解惑
